efb_telegram_master/slave_message.py

```diff
@@ -263,6 +263,10 @@ def get_slave_msg_dest(self, msg: EFBMsg) -> Tuple[str, Optional[TelegramChatID]
         msg_template = self.generate_message_template(msg, tg_chat, multi_slaves)
         self.logger.debug("[%s] Message is sent to Telegram chat %s, with header \"%s\".",
                           xid, tg_dest, msg_template)
+
+        if cache.get(tg_dest) and cache.get(tg_dest) != chat_uid:
```

Comment:
`chat_uid` here is guaranteed not `None`, the first clause of this `if` statement is redundant.

```suggestion
        if cache.get(tg_dest) != chat_uid:
```